datanode/api/event_observer.go

```diff
@@ -186,31 +188,33 @@ func observeEventsWithAck(
 	}
 }
 
-func recvEventRequest(
-	stream eventBusServer,
-) (*protoapi.ObserveEventBusRequest, error) {
-	readCtx, cfunc := context.WithTimeout(stream.Context(), 5*time.Second)
-	oebCh := make(chan protoapi.ObserveEventBusRequest)
-	var err error
+const defaultReqTimeout = time.Second * 5
```

Comment:
Const are usually at the top of the file, also would it make sense for this to be a config in the config file not saying it should, probably doesn't need to be.

Reply:
I'd move it into config only if that config would be kind of for general stream timeouts or whatever, and not just for this one case. Otherwise it would be overkill imo